test/PowerShellEditorServices.Test.Host/LanguageServerTests.cs

```diff
@@ -257,6 +258,97 @@ public async Task CompletesDetailOnCommandSuggestion()
             Assert.True(updatedCompletionItem.Documentation.Length > 0);
         }
 
+        [Fact]
+        public async Task CompletesDetailOnFilePathSuggestion()
+        {
+            string expectedPathSnippet;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                expectedPathSnippet = @".\TestFiles\CompleteFunctionName.ps1";
+            }
+            else
+            {
+                expectedPathSnippet = "./TestFiles/CompleteFunctionName.ps1";
+            }
+
+            // Change dir to root of this test project's folder
+            await this.SetLocationForServerTest(this.TestRootDir);
+
+            await this.SendOpenFileEvent(TestUtilities.NormalizePath("TestFiles/CompleteFunctionName.ps1"));
+
+            CompletionItem[] completions =
+                await this.SendRequest(
+                    CompletionRequest.Type,
+                    new TextDocumentPositionParams
+                    {
+                        TextDocument = new TextDocumentIdentifier
+                        {
+                            Uri = TestUtilities.NormalizePath("TestFiles/CompleteFunctionName.ps1")
+                        },
+                        Position = new Position
+                        {
+                            Line = 8,
+                            Character = 35
+                        }
+                    });
+
+            CompletionItem completionItem =
+                completions
+                    .FirstOrDefault(
+                        c => c.InsertText == expectedPathSnippet);
+
+            Assert.NotNull(completionItem);
+            Assert.Equal(InsertTextFormat.PlainText, completionItem.InsertTextFormat);
+        }
+
+        [Fact]
+        public async Task CompletesDetailOnFolderPathSuggestion()
+        {
+            string expectedPathSnippet;
+            InsertTextFormat insertTextFormat;
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                expectedPathSnippet = @"'.\TestFiles\Folder With Spaces$0'";
+                insertTextFormat = InsertTextFormat.Snippet;
+            }
+            else
+            {
+                expectedPathSnippet = @"./TestFiles/Folder\ With\ Spaces";
```

Comment:
Is this valid? On linux I get:

```
Get-ChildItem ./Folder\ With\ Spaces/  
Get-ChildItem : A positional parameter cannot be found that accepts argument 'Spaces/'.
At line:1 char:1
+ Get-ChildItem ./Folder\ With\ Spaces/
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (:) [Get-ChildItem], ParameterBindingException
+ FullyQualifiedErrorId : PositionalParameterNotFound,Microsoft.PowerShell.Commands.GetChildItemCommand
```

Reply:
Good catch, I must have been in Bash when I got that auto-completion of that folder.  I'll fix it.

Reply:
Fixed